requirements/default.txt

```diff
@@ -155,7 +155,7 @@ humanfriendly==4.12 \
 jsonschema==2.6.0 \
     --hash=sha256:000e68abd33c972a5248544925a0cae7d1125f9bf6c58280d37546b946769a08 \
     --hash=sha256:6ff5f3180870836cae40f06fa10419f557208175f13ad7bc26caa77beb1f6e02
-functools32==3.2.3-2 \
+functools32==3.2.3-2 ; python_version < '3.2' \
```

Comment:
This is a backported module. We'll need to do this sort of thing if we use any other backported modules. Having said that, I didn't hit other similar issues.